geometry_msgs/Pose start_pose
geometry_msgs/Pose goal_pose
geometry_msgs/Point[] target_points
float64[] start_joints
float64[] goal_joints
string target_id